mobile/src/screens/HomeScreen.tsx

```diff
@@ -65,58 +66,100 @@ export const HomeScreen: React.FC = () => {
   };
 
   return (
-
     <KeyboardAvoidingView
-      behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
+      behavior={Platform.OS === "ios" ? "padding" : "height"}
       style={{ flex: 1 }}
     >
       <ScrollView>
         {/* Orange box above the screen, in case the user scrolls past the top of the screen */}
-        <View style={styles.top}/>
+        <View style={styles.top} />
         {/* Starts at the top of the screen, containing the welcome videos and orange rounded background */}
         <View style={styles.heading}>
-          <SafeAreaView edges={['top']}>
-            <ScrollView horizontal pagingEnabled showsHorizontalScrollIndicator={false} style={styles.videos} contentContainerStyle={styles.videoChildContainer}>
-              {TOP_VIDEOS.map(video => (
-                <View key={video.url} style={styles.videoContainer}>
-                  <YoutubeVideo url={video.url} width={0.8 * width} height={9/16 * 0.8 * width} />
-                  <Text style={[TextStyles.heading2, styles.videoTitle]}>{video.title}</Text>
-                </View>
-              ))}
-            </ScrollView>
+          <SafeAreaView edges={["top"]}>
+            <OfflineIndicator variant="white" style={{ height: "60%" }}>
```

Comment:
There might be a better way to make the indicator fill up the height here instead of doing`height: "60%"`. Open to any ideas.

Reply:
![lgtm](https://i.imgur.com/xvWUf09.png)